### PR TITLE
Extensions - Parse Startup Parameters Config File

### LINKED ISSUE #959: Parse options from parameter files
**Is your enhancement related to a problem?**
Arma 3 launcher doesn't support custom command line arguments so -skipAcrePluginCopy can't be added through standard launcher

**Solution you'd like:**
Look for -par parameters and parse options in those files to check for -skipAcrePluginCopy.
https://community.bistudio.com/wiki/Startup_Parameters_Config_File

**Alternatives you've considered:**
Launching outside of the launcher breaks normal workflow.

**Additional context:**
When I'm working from my own ACRE build I have all dev related launch options in a parameter file so that I only have to enable a checkbox and unload my mods to work from it. -skipAcrePluginCopy was not detected when it was in a parameter file.

**Definition of Done:**
- ACRE can find and read parameter file format from post arma2 build 86060
- Old `class Arg` .par format won't be supported.
- Options are appended to the output of getcommandline so they appear as if they were on the process commandline.

### PR DESCRIPTION
**Pull request by ThymoNL**
**When merged this pull request will:**
- Closes #959
- Parse parameters from a [startup parameter file](https://community.bistudio.com/wiki/Startup_Parameters_Config_File#Pre_build_86060)
- Ignore old style parameter files which are broken on dedicated servers anyway
- Make it easier to parse command line arguments in the future (argv style)

### CODE REVIEW

**Comment by ThymoNL** [comment]
@Sniperhid Can you review this since you wrote the plugin copying code?

File: extensions/src/ACRE2Steam/CallExt_DllMain.cpp [hunk 1]
```diff
@@ -33,14 +33,58 @@ void __stdcall RVExtensionVersion(char *output, int outputSize) {
...
     GetModuleFileNameA(nullptr, moduleName, MAX_PATH);
     return std::string(moduleName);
 }
 
-inline std::string get_cmdline() {
-    return std::string(GetCommandLineA());
+std::vector<std::string> get_cmdline() {
+    std::vector<std::string> cmdline(split(GetCommandLineA(), ' '));
+
+    std::string par_opt("-par=");
```
Comment (TheMagnetar) [inline extensions/src/ACRE2Steam/CallExt_DllMain.cpp]:
```suggestion
    const std::string par_opt("-par=");
```

**Review by TheMagnetar — CHANGES_REQUESTED** [review]
just some minor comment

**Comment by Sniperhid** [comment]
Functionally looks okay, I shall defer to @TheMagnetar

**Review by TheMagnetar — APPROVED** [review]

**Pull request merged**